### ISSUE TITLE
Unsupported TF Select ops: AddV2, Mul, Pad

### ISSUE THREAD

**Issue by ARKAD97** [tensorflow/tensorflow#49658]
### System information

- OS Platform and Distribution (e.g., Linux Ubuntu 18.04):
- TensorFlow installation (pip package):
- TensorFlow library ( 2.5.0 ):

### Problem
I have an ONNX model, which itself is originated from pytorch. I'm trying to convert from ONNX format to tensorflow lite format without using Select ops.
First, I convert to TF SavedModel via onnx-tensorflow package, then I convert to TFLite with basic API:

```python
converter = tf.lite.TFLiteConverter.from_saved_model( name )
converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS]
```

I get the following errors:

```bash
...
<unknown>:0: note: loc("PartitionedCall"): called from
<unknown>:0: error: loc(callsite(callsite("onnx_tf_prefix_Add_65@__inference___call___373" at "PartitionedCall@__inference_signature_wrapper_435") at "PartitionedCall")): 'tf.AddV2' op is neither a custom op nor a flex op
<unknown>:0: note: loc("PartitionedCall"): called from
<unknown>:0: error: failed while converting: 'main': 
Some ops are not supported by the native TFLite runtime, you can enable TF kernels fallback using TF Select. See instructions: https://www.tensorflow.org/lite/guide/ops_select 
TF Select ops: AddV2, Mul, Pad
Details:
        tf.AddV2
        tf.AddV2 {device = ""}
        tf.Mul {device = ""}
        tf.Pad {device = ""}
```

I've also had problem with SoftMax operator, but I've solved the issue. It turned out, that TFLite version of [softmax operator](https://www.tensorflow.org/api_docs/python/tf/nn/softmax) doesn't support axis parameter, so I removed it and now converter is able to use non-flex softmax.
All modifications I do on ONNX graph, before converting to Tensorflow

What could be wrong with other operators: AddV2, Mul, Pad ?

### What I tried
I've noticed that [tfl.mul](https://www.tensorflow.org/mlir/tfl_ops#tflmul_tflmulop) docs don't tell anything about broadcasting compared to [tf.mul](https://www.tensorflow.org/mlir/tf_ops#tfmul_tfmulop). So my hypothesis was that non-flex tflite can multiply only tensors of the same shape, but can't multiply tensor by scalar. So I've replaced [Constant](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Constant) with [Shape](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Shape) -> [Cast](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Cast) (to float32) -> [ConstantOfShape](https://github.com/onnx/onnx/blob/master/docs/Operators.md#ConstantOfShape) operators, but I still get the same errors.

I've also tried to convert all tensors with datatype int64 to int32 - no success.

Any suggestions? What do I have to pay attention first, when I get such select ops issues?

**Comment by abattery** [comment]
Please consider using the Select TF op option. https://www.tensorflow.org/lite/guide/ops_select

**Comment by abattery** [comment]
If you can fix the input shapes of AddV2 and Mul operators or lower the dimensions, those math operators can be replaced with the TFLite operators.

If you can lower the input dimension of the Pad operator (input dim <= 4), the converter will put the TFLite Pad op.

**Comment by ARKAD97** [comment]
> 
> 
> If you can fix the input shapes of AddV2 and Mul operators or lower input dimensions regardless of broadcasting support (e.g., input dim <= 4), those math operators can be replaced with the TFLite operators.
> 
> If you can lower the input dimension of the Pad operator (input dim <= 4), the converter will put the TFLite Pad op.

Thanks for solution! I'll experiment and try to reshape tensor before and after AddV2 and Mul operator (probably next week).
I'll write here when I succeed.

**Comment by ARKAD97** [comment]
So, I did the following things:
1. Made all inmput dimensions in ONNX graph fixed, that fixed AddV2 and Mul operators
2. Lowered dimension of input for Pad operator

And this helped me to get rid of flex operators
